docs-source/content/userguide/managing-domains/_index.md

```diff
@@ -83,22 +83,22 @@ The following features are **not** certified or supported in this release:
 * Multitenancy
 * Production redeployment
 
-Please consult My Oracle Support Doc ID 2349228.1 for up-to-date information about the features of WebLogic Server that are supported in Kubernetes environments.
+For up-to-date information about the features of WebLogic Server that are supported in Kubernetes environments, see My Oracle Support Doc ID 2349228.1.
 
 
 ### Creating and managing WebLogic domains
 
 You can locate a WebLogic domain either in a persistent volume (PV) or in a Docker image.
-For examples of each, see the [WebLogic Kubernetes Operator samples]({{< relref "/samples/simple/domains/_index.md" >}}).
+For examples of each, see the [WebLogic Server Kubernetes Operator samples]({{< relref "/samples/simple/domains/_index.md" >}}).
 
 If you want to create your own Docker images, for example, to choose a specific set of patches or to create a domain
-with a specific configuration and/or applications deployed, then you can create the domain custom resource
+with a specific configuration or applications deployed, then you can create the domain custom resource
 manually to deploy your domain.  This process is documented in [this
 sample]({{< relref "/samples/simple/domains/manually-create-domain/_index.md" >}}).
 
 ### Modifying domain configurations
 
-You can modify the WebLogic domain configuration for both the "domain in persistent volume" and the "domain in image" options before deploying a domain resource:
+You can modify the WebLogic domain configuration for both Domain in PV and Domain in Image before deploying a domain resource:
```

Comment:
Modifying domain configurations

You can modify the WebLogic domain configuration for Domain in PV, Domain in Image, and Model in Image before deploying a domain resource:

When the domain is in a persistent volume, you can use WLST or WDT to change the configuration.

For Domain in Image and Domain in PV you can use [configuration overrides]({{< relref "/userguide/managing-domains/configoverrides/_index.md" >}}). Configuration overrides allow changing a configuration without modifying its original config.xml or system resource XML files, and also support parameterizing overrides so that you can inject values into them from Kubernetes secrets. For example, you can inject database user names, passwords, and URLs that are stored in a secret.

For Model in Image you use [model updates]({{<relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}).

Reply:
updated, please check

Reply:
looks good